locations/spiders/lidl_de.py

```diff
@@ -18,67 +13,67 @@
     "Sa": "Sa",
     "So": "Su",
 }
-
-
 class LidlDESpider(scrapy.Spider):
     name = "lidl_de"
-    item_attributes = {"brand": "Lidl", "brand_wikidata": "Q151954"}
+    item_attributes = {"brand": "Lidl", "brand_wikidata": "Q151954", "country": "DE"}
     allowed_domains = ["lidl.de"]
     handle_httpstatus_list = [404]
-    start_urls = ["https://www.lidl.de/de/filialsuche/s940"]
+    start_urls = ["https://www.lidl.de/f/"]
 
-    def parse_hours(self, store_hours):
+    def parse_hours(self, hours):
         opening_hours = OpeningHours()
-        if store_hours is None:
-            return
 
-        for store_day in store_hours:
-            day, hours = store_day.split(" ")
-            match = re.match(r"(\d\d:\d\d)-(\d\d:\d\d)", hours)
-            if match:
+        for item in hours:
+            if (item.split()):
+                day = DAY_MAPPING[item.split()[0]]
```

Comment:
This is failing on [this](https://www.lidl.de/f/nuernberg) "15.08. 07:00-20:00". Maybe wrap line 28+ in a try statement?

Reply:
I've fixed this in https://github.com/alltheplaces/alltheplaces/pull/3764/commits/5224d551d98f50629605226da38d436194a399c3